lib/ioc-rc

```diff
@@ -543,10 +543,12 @@ __rc_jails () {
         local name="$(zfs get -H -o value org.freebsd.iocage:host_hostuuid \
                     $jail)"
         local boot="$(zfs get -H -o value org.freebsd.iocage:boot $jail)"
+        local _jail_hostid="$(zfs get -H -o value org.freebsd.iocage:hostid \
+                        $jail)"
         local priority="$(zfs get -H -o value org.freebsd.iocage:priority \
                         $jail)"
 
-        if [ "$boot" == "on" ] ; then
+        if [ "$boot" == "on" ] && [ "$_jail_hostid" == "$hostid" ] ; then
```

Comment:
Interesting, I had not seen that. The only reason I was doing it was I found it easier to read. It also reduced a few characters on each line, so that was a bonus. I don't know about you but this:

```
if [ "$a" == "on" ] && [ "$b" == "off" ] || [ "$c" == "yes" ] ; then
```
Looks much uglier and more difficult to read then:

```
if [ "$a" == "on" -a "$b" == "off" -o "$c" == "yes" ] ; then
```